packages/tokens/build.js

```diff
@@ -217,7 +223,12 @@ StyleDictionary.registerFormat({
       {},
     )
 
-    return JSON.stringify(sortTokensByKey(tokens), undefined, 2) + `\n`
+    const sortedTokens =
+      dictionary.allTokens?.[0].attributes?.category == 'spacing'
```

Comment:
```suggestion
      dictionary.allTokens?.[0].attributes?.category === 'spacing'
```
?

Probably fine, but not sure we (or the app) ever uses JS's `==`.